### ISSUE TITLE
inlineCustomMsgs not correctly added to websocket event

### ISSUE THREAD

**Issue by liamaharon** [blocknative/assist#248]
Encountered this while adding websocket-event validation on the server. 

If I add some inlineCustomMsgs to my `.send` call, `event.inlineCustomMsgs` is received as `{}` on the backend. I would have expected the object to contain some info about the inline custom messages.

If I don't add inlineCustomMsgs to my `.send` calls, `event.inlineCustomMsgs` is `undefined`.

@cmeisl @aaronbarnardsound since this is a very non-breaking bug and Aaron will be rewriting the notifications library soon, is it a waste of time to debug? Otherwise let me know and I can spend some time digging into it.

(I'm using web3 v1)

**Comment by aaronbarnardsound** [comment]
My guess is that the object is empty due to it being stringified before being logged and it isn't serializable? We should probably just take it off the event object that gets logged to the server as it isn't necessary to log it anyway.

**Comment by liamaharon** [comment]
Yeah you're right, the values in the objects are all functions so not serializable. Makes sense to me to take them off the event object
